template/registry/number_test.go

```diff
@@ -11,66 +11,66 @@ func Test_Number(t *testing.T) {
 
 	// decimal
 
-	assert := assertFmt(t, numberRegistryFunc, nil, language.Latvian)
+	assert := assertFormat(t, numberRegistryFunc, nil, language.Latvian)
 	assert(-0.1234, "-0,123")
 	assert(0, "0")
 	assert(0.1234, "0,123")
 
-	assert = assertFmt(t, numberRegistryFunc, map[string]any{"signDisplay": "auto"}, language.AmericanEnglish)
+	assert = assertFormat(t, numberRegistryFunc, map[string]any{"signDisplay": "auto"}, language.AmericanEnglish)
 	assert(-0.15, "-0.15")
 	assert(0, "0")
 	assert(0.15, "0.15")
 
-	assert = assertFmt(t, numberRegistryFunc, map[string]any{"signDisplay": "always"}, language.AmericanEnglish)
+	assert = assertFormat(t, numberRegistryFunc, map[string]any{"signDisplay": "always"}, language.AmericanEnglish)
 	assert(-0.15, "-0.15")
 	assert(0, "+0")
 	assert(0.15, "+0.15")
 
-	assert = assertFmt(t, numberRegistryFunc, map[string]any{"signDisplay": "exceptZero"}, language.AmericanEnglish)
+	assert = assertFormat(t, numberRegistryFunc, map[string]any{"signDisplay": "exceptZero"}, language.AmericanEnglish)
 	assert(-0.15, "-0.15")
 	assert(0, "0")
 	assert(0.15, "+0.15")
 
-	assert = assertFmt(t, numberRegistryFunc, map[string]any{"signDisplay": "never"}, language.AmericanEnglish)
+	assert = assertFormat(t, numberRegistryFunc, map[string]any{"signDisplay": "never"}, language.AmericanEnglish)
 	assert(-0.15, "0.15")
 	assert(0, "0")
 	assert(0.15, "0.15")
 
-	assert = assertFmt(t, numberRegistryFunc, map[string]any{"maximumFractionDigits": 1}, language.AmericanEnglish)
+	assert = assertFormat(t, numberRegistryFunc, map[string]any{"maximumFractionDigits": 1}, language.AmericanEnglish)
 	assert(0.15, "0.1")
```

Comment:
No rounding up?

Reply:
No. MF2 does not describe the rounding strategy. However, the `number` function borrows ideas from EcmaScript - [Intl.NumberFormat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat).

In Javascript, it does round up by default:

```javascript
console.log(
  new Intl.NumberFormat('en', { maximumFractionDigits: 2 }).format(
    6.785,
  ),
)
// prints "6.79"
```

Reply:
Golang does not round up by default.

Reply:
Leaving it as is, the MF2 WG does not define the implementation/formatting details. The options are hints for the implementation.

[More](https://github.com/unicode-org/message-format-wg/blob/1dc84e648a6f98d74ac62306abaacc0bed8e4fc5/meetings/2024/notes-2024-02-14.md)